src/main/java/school/lemon/changerequest/java/Employees/Programmer.java

```diff
@@ -0,0 +1,17 @@
+package school.lemon.changerequest.java.Employees;
+
+/**
+ * Created by lbrdev on 04.01.2017.
+ * Project: oop.pr1
+ */
+public class Programmer extends Employee {
+
+    public Programmer(String name) {
+        super(name);
+    }
+
+    @Override
+    public double calculateSalary() {
```

Comment:
Why do you override method with exactly the same implementation? I'd prefer to make base method abstract.